shared/utils/add_stig_references.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python2
+
+try:
+    from xml.etree import cElementTree as etree
+except ImportError:
+    import cElementTree as etree
+
+import re
+import sys
+import argparse
+
+parser = argparse.ArgumentParser(
+    description='Add Stig references to XCCDF files.')
+parser.add_argument("reference", help="Path to DISA STIG Reference XCCDF")
+parser.add_argument("destination", help="Path to XCCDF file")
+args = parser.parse_args()
+
+reference = args.reference
+destination = args.destination
+
+xccdf_namespace = "http://checklists.nist.gov/xccdf/1.1"
+stig_href = 'http://iase.disa.mil/stigs/Pages/stig-viewing-guidance.aspx'
+stig_references_beginning = 'http://iase.disa.mil/stigs/'
+
+try:
+    reference_root = etree.parse(reference)
+except IOError as exception:
+    print 'INFO: DISA STIG Reference file not found for this platform'
+    sys.exit(0)
+
+reference_rules = reference_root.findall('.//{'+xccdf_namespace+'}Rule')
+
+dictionary = {}
+
+for r in reference_rules:
+    dictionary[r.find('.//{'+xccdf_namespace+'}version').text] = r.get('id')
+
+target_root = etree.parse(destination)
+target_rules = target_root.findall('.//{'+xccdf_namespace+'}Rule')
```

Comment:
All instances of `{'+xccdf_namespace+'}` in the code to `'{%s}' % xccdf_namespace`